11/umbraco-cms/reference/notifications/hot-vs-cold-restarts.md

```diff
@@ -27,6 +27,12 @@ Below is a list of some of the more common reasons:
 * Examine indexes need to rebuild - for large sites, this can take some time.
 * The custom code on the website is not optimized and does not live up to .NET standards.
 
+## In Memory Auto
+
+Another factor that can slow down time to first page load is the In Memory Auto models builder setting. Having this setting enabled will result in the first page load being slower. The reason for this is that when the first page is requested, the strongly typed models needs to be compiled and loaded in. 
+
+This is, however, less noticable on consecutive restarts, a bit like hot and cold restarts, since the compiled models will be cached on disk, so they don't need to be recompiled untill the models change again.
```

Comment:
Long sentence here 😅 
Not sure I did it right - can you please double-check that my changes didn't remove any of the meaning?

```suggestion
This is, however, less noticeable on consecutive restarts, a bit like hot and cold restarts. Since the compiled models will be cached on disk, they don't need to be recompiled until the models change again.
```

Reply:
Looks good to me, thanks 👍